addons/mail/models/mail_thread.py

```diff
@@ -2746,3 +2746,12 @@ def _get_mail_redirect_suggested_company(self):
         if 'company_id' in self:
             return self.company_id
         return False
+
+    def read_attachments(self):
+        domain = [
+            ('res_id', '=', self.id),
+            ('res_model', '=', self._name)
+        ]
+        fields = ['id', 'name', 'datas_fname', 'mimetype']
```

Comment:
datas_fname was not passed in the original js?
```
fields: ['id', 'name', 'mimetype'],
```

Reply:
done